modular_skyrat/modules/stock_market/code/articles.dm

```diff
@@ -0,0 +1,125 @@
+/proc/consonant()
+	return pick("B","C","D","F","G","H","J","K","L","M","N","P","Q","R","S","T","V","W","X","Y","Z")
+
+/proc/vowel()
+	return pick("A", "E", "I", "O", "U")
+
+/proc/ucfirst(var/S)
+	return "[uppertext(ascii2text(text2ascii(S, 1)))][copytext(S, 2)]"
+
+/proc/ucfirsts(var/S)
+	var/list/L = splittext(S, " ")
+	var/list/M = list()
+	for (var/P in L)
+		M += ucfirst(P)
+	return jointext(M, " ")
```

Comment:
Look at all these *single letter variables*

Reply:
The god and legend prodirus went out of his way to uncancer this all to actual meanings